include/ei_ncs.rst

```diff
@@ -0,0 +1,136 @@
+.. _ei_ncs:
+
+Edge Impulse NCS library
+########################
+
+The Edge Impulse NCS library implements running a machine learning model on embedded device.
+The machine learning model is trained using Edge Impulse web tools.
+
+Overview
+********
+
+The `Edge Impulse`_ is a development platform that can be used to enable machine learning on edge devices.
+You can use the platform to collect data from sensors, train machine learning model and deploy it to an embedded device.
+The `Edge Impulse studio`_ provides a set of web-based tools that can be used for that purpose.
+
+The Edge Impulse NCS library is a wrapper around the C++ library deployed in Edge Impulse studio.
+The wrapper:
+
+* Buffers input data for the machine learning model.
+* Runs the machine learning model in a separate thread.
+* Provides results via a dedicated callback.
+
+Configuration
+*************
+
+By default, the library is not used in any sample.
```

Comment:
remove this line

Reply:
done

Reply:
I meant the text "By default, the library is not used in any sample." :)

Reply:
Added: `Complete the following steps to enable and configure the Edge Impulse NCS library.` We need to avoid empty paragraphs.